src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import Overlay from 'react-overlays/lib/Overlay';
+import Transition from 'react-overlays/lib/Transition';
+import styled, { injectGlobal } from 'styled-components';
+import { colors } from '../../theme';
+
+injectGlobal(`
```

Comment:
I didn't think of that.  No, I definitely don't want those classes injected every time.  I was trying to avoid using an imported less file, and the Transition component wants classes. On the react-overlay website, they say "You don't even need to use class based css transitions if you don't want to (but it is easiest)", but they offer no example of how to do this.  I'd prefer that if I can figure out how.